src/current/v23.1/cockroach-start.md

```diff
@@ -384,21 +384,35 @@ In this example we will start a multi-node [local cluster]({% link {{ page.versi
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    cockroach start --locality=region=us-east1,zone=us-east-1a --insecure --store=/tmp/node0 --listen-addr=localhost:26257 --http-port=8888  --join=localhost:26257,localhost:26258,localhost:26259 --background
+    cockroach start --locality=region=us-east1,zone=us-east-1a \
+                    --insecure --store=/tmp/node0 \
+                    --listen-addr=localhost:26257 \
+                    --http-port=8888 \
+                    --join=localhost:26257,localhost:26258,localhost:26259
     ~~~
 
 1. Start a node in the `us-west1` region:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    cockroach start --locality=region=us-west1,zone=us-west-1a --insecure --store=/tmp/node2 --listen-addr=localhost:26259 --http-port=8890  --join=localhost:26257,localhost:26258,localhost:26259 --background
+    cockroach start --locality=region=us-west1,zone=us-west-1a \
```

Comment:
After the first line in this command, the subsequent lines are indented heavily when rendered — not sure if this is intentional. (same for both versions)